.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,65 @@
+name: code-coverage
+
+on:
+  push:
+    branch: sima-history
+  workflow_dispatch:
+  pull_request:
```

Comment:
I may be missing something, but shouldn't this have caused the tests to be triggered on this PR?

Reply:
oh i think i just haven't set up github actions for my fork...

Reply:
Yup!  In my experience, these won't run on this fork until it gets merged unfortunately.  ☹️